### ISSUE TITLE
[BUG] Keyguard not shown in Android 15 Accessibility Apps

### ISSUE THREAD

**Issue by mattdale77** [AChep/keyguard-app#1085]
**Describe the bug**
In Android 15, under Settings -> Accessibility & Convenience -> Accessibility -> Downloaded Apps, keyguard is not shown in order to enable the password prompt so it does not work for apps which do not use the built in autofill functionality and possibly many more

**To Reproduce**
Install Keyguard on Android 15 and try to get it working for password prompts

**Expected behavior**
Keyguard should be shown in keyboard smart bar or a hovering window on password field

**Screenshots**

I have taken one and I'll try to add in a comment. I can't see how to upload

**Smartphone (please complete the following information):**
 - Device: Oppo Find X5 Pro & OnePlus 13
 - OS: Android 15
 - Browser: IronFox (but the issue mostly presents in other apps
 - App version 1.15.0-127580007

**Additional context**

Quite a few apps don't show the keyguard prompt so I can't autifill. One example is Twitch but I can provide others

**Comment by mattdale77** [comment]
![Image](https://github.com/user-attachments/assets/6ae0ab38-83d4-4bf8-932c-84ad4a1aadbb)
